datadog/resource_datadog_logs_integration_pipeline.go

```diff
@@ -0,0 +1,68 @@
+package datadog
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/zorkian/go-datadog-api"
+	"strings"
+)
+
+func resourceDatadogLogsIntegrationPipeline() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDatadogLogsIntegrationPipelineCreate,
+		Update: resourceDatadogLogsIntegrationPipelineUpdate,
+		Read:   resourceDatadogLogsIntegrationPipelineRead,
+		Delete: resourceDatadogLogsIntegrationPipelineDelete,
+		Exists: resourceDatadogLogsIntegrationPipelineExists,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"is_enabled": {Type: schema.TypeBool, Optional: true},
+		},
+	}
+}
+
+func resourceDatadogLogsIntegrationPipelineCreate(d *schema.ResourceData, meta interface{}) error {
+	return fmt.Errorf("cannot create an integration pipeline, try import it")
+}
+
+func resourceDatadogLogsIntegrationPipelineRead(d *schema.ResourceData, meta interface{}) error {
+	ddPipeline, err := meta.(*datadog.Client).GetLogsPipeline(d.Id())
+	if err != nil {
+		return err
+	}
+	if err := d.Set("is_enabled", ddPipeline.GetIsEnabled()); err != nil {
+		return err
+	}
+	return nil
+}
+
+func resourceDatadogLogsIntegrationPipelineUpdate(d *schema.ResourceData, meta interface{}) error {
+	var ddPipeline datadog.LogsPipeline
+	ddPipeline.SetIsEnabled(d.Get("is_enabled").(bool))
+	client := meta.(*datadog.Client)
+	updatedPipeline, err := client.UpdateLogsPipeline(d.Id(), &ddPipeline)
+	if err != nil {
+		return fmt.Errorf("error updating logs pipeline: (%s)", err.Error())
+	}
+	d.SetId(*updatedPipeline.Id)
+	return resourceDatadogLogsIntegrationPipelineRead(d, meta)
+}
+
+func resourceDatadogLogsIntegrationPipelineDelete(d *schema.ResourceData, meta interface{}) error {
+	return nil
```

Comment:
Does this still get removed from the state?

Reply:
yes, it should be removed from the state but not from backend.

Reply:
ok great !